subprojects/android-test/test-app-kaspresso/src/androidTest/kotlin/com/avito/android/kaspressoui/test/screen/MainScreen.kt

```diff
@@ -0,0 +1,23 @@
+package com.avito.android.kaspressoui.test.screen
+
+import com.agoda.kakao.text.KButton
+import com.agoda.kakao.text.KTextView
+import com.avito.android.kaspressoui.MainActivity
+import com.avito.android.kaspressoui.R
+import com.kaspersky.kaspresso.screens.KScreen
+import com.kaspersky.kaspresso.testcases.core.testcontext.BaseTestContext
+
+object MainScreen : KScreen<MainScreen>() {
+
+    override val layoutId: Int? = R.layout.activity_main
```

Comment:
It's good. We can try an impact analysis on Kaspresso page objects.
@dsvoronin , cc

Reply:
@eugene-krivobokov I suggest to create a task and put it in our Avokado backlog. What do you think?